content/blog/2016-01-26-one-year-of-open-prometheus-development.md

```diff
@@ -0,0 +1,139 @@
+SoundCloud to work on Prometheus. He has since become the most active developer
+on the project and we have him to thank for major new features such as
+generalized service discovery support, runtime-reloadable configurations, new
+powerful query language features, a custom-built query parser, and so much
+more. He is currently working on the new beta rewrite of the
+[Alertmanager](https://github.com/prometheus/alertmanager).
+
+Finally, we have been honored to be recognized and adopted by major players in
+the industry. [Google](https://www.google.com) is now instrumenting its open-source
+container management system [Kubernetes](http://kubernetes.io/) natively with
+Prometheus metrics. [CoreOS](https://coreos.com/) is picking it up for
+[etcd](https://coreos.com/etcd/)'s monitoring as well. [DigitalOcean](https://www.digitalocean.com/) is betting on Prometheus for their
+internal monitoring. By now, the list of companies using Prometheus in one way
+or another has become too long to mention: [Google](https://www.google.com),
+[CoreOS](https://coreos.com/), [Docker](https://docker.com),
+[Boxever](http://www.boxever.com/),
+[DigitalOcean](https://www.digitalocean.com/), [Financial Times](http://www.ft.com/),
+[Improbable](http://improbable.io/), [KPMG](https://kpmg.com), and many more.
+Even the world's largest digital festival,
+[DreamHack](https://www.dreamhack.se), has [used
+Prometheus](http://prometheus.io/blog/2015/06/24/monitoring-dreamhack/) to keep
+tabs on their network infrastructure in 2015, and
+[FOSDEM](https://fosdem.org/2016/) will do so in 2016.
+
+The widely popular dashboard builder [Grafana](http://grafana.org/) also added
+native Prometheus backend support in [version
+2.5](http://grafana.org/blog/2015/10/28/Grafana-2-5-Released.html). Since
+people all around the world are already using and loving Grafana, we are going
+to focus on improving Grafana's Prometheus integration and will likely invest
+less energy in our own dashboard builder
+[PromDash](https://github.com/prometheus/promdash) in the future.
+
+On a funnier note, 2015 has also been the year in which Prometheus's query
+language was proven to be Turing complete by
+[Brian Brazil](https://github.com/brian-brazil) by implementing [Conway's Game of Life
+in PromQL](http://www.robustperception.io/conways-life-in-prometheus/).
+
+## The road ahead
+
+Both personally and technically, we are really excited about what has happened
+last year in Prometheus-land. We love the opportunity to provide the world with
+a powerful new approach to monitoring, especially one that is much better
+suited towards modern cloud- and container-based infrastructures than
+traditional solutions. We are also very grateful to all contributors and
+hope to continuously improve Prometheus for everyone.
+
+Although Prometheus is relatively mature by now, we have a list of major goals
+we want to tackle in 2016. The highlights will be stabilizing the new
```

Comment:
This reads as though it isn't already stable (which it as as far as we know), I'd say "polishing" instead.